### ISSUE TITLE
Buttons in inventory window visible during shopping

### ISSUE THREAD

**Issue by x-qq** [calref/cboe#233]
![zh0V](https://user-images.githubusercontent.com/55463047/72582951-248f5800-38ed-11ea-85c0-70959250bc46.png)

The Use, Give, Drop buttons were not visible in the original game.

**Comment by wrldwzrd89** [comment]
This looks like a simplification gone awry - the reason the Use/Give/Drop buttons weren't orignally displayed while shopping is because they don't make sense to use in that context.

**Comment by CelticMinstrel** [comment]
Right, and they _still_ don't make sense in that context, so they still shouldn't be shown.
